packages/visualizations/src/components/Map/utils.ts

```diff
@@ -36,15 +49,15 @@ export const colorShapes = (geoJson, values, colorsScale, emptyValueColor) => {
         scale = chroma.scale([colorMin, colorMax]).domain([min, max]);
     }
 
-    const dataMapping = {};
+    const dataMapping: any = {};
```

Comment:
I tried the reduce, but it ends up being a bit more verbose because the initial value `{}` also needs the same type etc.
But I completely missed that I could have an object where both the key and value can be typed, so I replaced the `any` with the inlined type you suggested!